tests/main/interfaces-audio-playback-record/task.yaml

```diff
@@ -0,0 +1,112 @@
+summary: Ensure that the audio-playback/record interface works
+
+# Only classic Ubuntu has the pulseaudio mediation patches
+systems: [ ubuntu-1*-*64 ]
+
+environment:
+    PLAY_FILE: "/snap/test-snapd-audio-record/current/usr/share/sounds/alsa/Noise.wav"
+    # today, only 19.04 and 19.10 have a mediating pulseaudio
+    EXFAIL: "ubuntu-14 ubuntu-16 ubuntu-18"
```

Comment:
Nitpick: based on how this is used I think it would be simpler if it was the full regular expression matching the expected failures: `"ubuntu-1[468]"`. This would remove the whitespace trickiness, the loop and everything below.

Reply:
Done